izumi-reflect/izumi-reflect/src/main/scala/izumi/reflect/macrortti/LightTypeTag.scala

```diff
@@ -353,6 +353,18 @@ object LightTypeTag {
     LightTypeTag(ref, mergedBasesDB, mergedInheritanceDb)
   }
 
+  def wildcardType(lowTag: LightTypeTag, highTag: LightTypeTag): LightTypeTag = {
+    val ref = LightTypeTagRef.WildcardReference(Boundaries.Defined(lowTag.ref.asInstanceOf[AbstractReference], highTag.ref.asInstanceOf[AbstractReference]))
```

Comment:
```suggestion
    val ref = LightTypeTagRef.WildcardReference(Boundaries.Defined(lowTag.ref match { case r: AbstractReference => r }, highTag.ref match { case r: AbstractReference => r }))
```
As in LightTypeTag.refinedType above